lib/src/dart_dev_cli.dart

```diff
@@ -182,9 +182,9 @@ Future _run(List<String> args) async {
 
   reporter.log('');
   if (result.successful) {
-    reporter.success(result.message, shout: true);
+    reporter.success(result.message ?? '', shout: true);
```

Comment:
I'm a little concerned/unclear what empty string will accomplish if result.message is null.

Reply:
`isNotEmpty` threw an exception if `result.message` is null here: https://github.com/Workiva/dart_dev/blob/master/lib/src/reporter.dart#L102

I'll push a commit to fix this there though. I'm still going to set it to an empty string if null so that it doesn't print "null" in the terminal when there's no `result.message`.

Reply:
`isNotEmpty` threw here: https://github.com/Workiva/dart_dev/blob/master/lib/src/reporter.dart#L102 

I will push a commit to fix it up farther down the stack though.